metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/2.3.3/configuration/elastic-site.xml

```diff
@@ -189,7 +189,12 @@
     </property>
     <property>
         <name>network_host</name>
-        <value>"_lo:ipv4_","_eth0:ipv4_"</value>
-        <description>Network interface(s) ES will bind to within each node. Confirm names via ifconfig.  Should have entries for primary external and loopback interfaces, with :ipv4 annotation and quote marks around each entry: "_lo:ipv4_","_eth0:ipv4_"</description>
+        <value>[ _local_ ]</value>
+        <description>Network interface(s) ES will bind to within each node. Use [ _local_ ] for single-node install; change to [ _local_, _site_ ] (including square brackets) for typical cluster install. See https://www.elastic.co/guide/en/elasticsearch/reference/2.3/modules-network.html for ES documentation.</description>
+    </property>
+    <property>
+        <name>network_publish_host</name>
```

Comment:
The only potential issue I see is with "network_publish_host". Ambari treats that as a required field, but it is not populated by default.  So I cannot change any other ES configuration value unless I define this field to something.  

Is this what you would expect? Is there some safe default that we can set this to instead of being empty?  

![image](https://cloud.githubusercontent.com/assets/2475409/25810236/7572ef4c-33dd-11e7-8f4d-cb2f32429dcb.png)